tur/stern/build.sh

```diff
@@ -0,0 +1,44 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/stern/stern
+TERMUX_PKG_DESCRIPTION="⎈ Multi pod and container log tailing for Kubernetes -- Friendly fork of https://github.com/wercker/stern"
```

Comment:
Ahh.. It seems that this description is too long

Reply:
ahhh ic 🤣, ok i'll fix it, maybe it should be just "Multi pod and container log tailing for Kubernetes"?